include/oneapi/dpl/async

```diff
@@ -19,13 +19,26 @@
 #include "oneapi/dpl/internal/common_config.h"
 #include "oneapi/dpl/pstl/onedpl_config.h"
 
-#include "oneapi/dpl/internal/async_extension_defs.h"
+#if _ONEDPL_BACKEND_SYCL
+#    if !_ONEDPL_ASYNC_FORWARD_DECLARED
+#        include "oneapi/dpl/internal/async_extension_defs.h"
+#        define _ONEDPL_ASYNC_FORWARD_DECLARED 1
+#    endif // !_ONEDPL_ASYNC_FORWARD_DECLARED
 
-#if _ONEDPL_EXECUTION_POLICIES_DEFINED
+#    if _ONEDPL_EXECUTION_POLICIES_DEFINED
 // If <execution> has already been included, pull in implementations
```

Comment:
I'd recommend to indent the comment same as `include` (ignoring the # sign).

Reply:
Fixed.

Reply:
But now we have clang format error here:

```
--- a/include/oneapi/dpl/async
+++ b/include/oneapi/dpl/async
@@ -26,7 +26,7 @@
 #    endif // !_ONEDPL_ASYNC_FORWARD_DECLARED
 
 #    if _ONEDPL_EXECUTION_POLICIES_DEFINED
-         // If <execution> has already been included, pull in implementations
+// If <execution> has already been included, pull in implementations
 #        include "oneapi/dpl/internal/async_impl/async_utils.h"
 #        include "oneapi/dpl/internal/async_impl/async_impl.h"
 #        include "oneapi/dpl/internal/async_impl/glue_async_impl.h"
Error: Process completed with exit code 1.
```

Reply:
I would say we should ignore this, as fixing it would make readability worse

Reply:
OK.